src/url-loader/package-url-resolver.ts

```diff
@@ -91,6 +125,44 @@ export class PackageUrlResolver extends UrlResolver {
       from = this.packageUrl;
       to = fromOrTo;
     }
+    return this.relativeImpl(from, to);
```

Comment:
The above unchanged part could benefit from some destructuring, imo:

```ts
const [from, to] = (maybeTo !== undefined) ? [fromOrTo, maybeTo] : [this.packageUrl, fromOrTo];
```

Reply:
+1